pkgs/development/compilers/ponyc/default.nix

```diff
@@ -24,32 +24,31 @@ stdenv.mkDerivation (rec {
   };
 
   ponygbenchmark = fetchurl {
-    url = "https://github.com/google/benchmark/archive/v1.5.0.tar.gz";
-    sha256 = "06i2cr4rj126m1zfz0x1rbxv1mw1l7a11mzal5kqk56cdrdicsiw";
-    name = "v1.5.0.tar.gz";
+    url = "https://github.com/google/benchmark/archive/v1.5.2.tar.gz";
```

Comment:
This version number should be substituted together with ...

Reply:
This no longer exists after moving to fetchFromGitHub